build/tasks/compare_size.mjs

```diff
@@ -73,6 +74,16 @@ function compareSizes( existing, current, padLength ) {
 	return chalk.green( `${delta}`.padStart( padLength ) );
 }
 
+function sortBranches( a, b ) {
+	if ( a === lastRunBranch ) {
+		return 1;
+	}
+	if ( b === lastRunBranch ) {
+		return -1;
+	}
+	return a.localeCompare( b );
```

Comment:
`localeCompare` may not be stable across environments. Adding two more checks for `>` & `<` would be a bit safer.